**Linux** [Build 9665](https://ci.wildfly.org/viewLog.html?buildId=17689&buildTypeId=WFPR) outcome was **FAILURE** using a merge of ecc4e54018a61c314e662a91d35f134db247c9e4
Summary: Tests failed: 7 (7 new), passed: 3355, ignored: 194 Build time: 00:57:35
### Failed tests
```
org.jboss.as.test.xts.wsat.client.ATTestCase: org.jboss.as.test.xts.annotation.client.CompensatableTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"compensatable-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"compensatable-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"compensatable-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"compensatable-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
01:51:57,435 WARNING [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Bundles path is deprecated and no longer used.
01:51:57,436 INFO  [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Starting container with: [/usr/java/latest/bin/java, -D[Standalone], -Dorg.jboss.ejb.client.wildfly-testsuite-hack=true, -Xmx512m, -XX:MetaspaceSize=128m, -Djboss.dist=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/../../../dist/target/wildfly-10.1.0.Final-SNAPSHOT, -Djava.net.preferIPv4Stack=false, -Djava.net.preferIPv6Addresses=true, -server, -Dts.timeout.factor=100, -Dnode0=::1, -Dnode1=::1, -Dmcast=ff13::1, -Dmcast.ttl=0, -Djbossas.ts.submodule.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts, -Djbossas.ts.integ.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/.., -Djbossas.ts.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/../.., -Djbossas.project.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/../../.., -Djboss.inst=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas, -Dorg.jboss.byteman.verbose, -Djboss.modules.system.pkgs=org.jboss.byteman, -Dorg.jboss.byteman.transform.all, -javaagent:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/lib/byteman.jar=listener:true, -ea, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas, -Dorg.jboss.boot.log.file=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/log/server.log, -Dlogging.configuration=file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/configuration/logging.properties, -jar, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/jboss-modules.jar, -mp, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/modules:/opt/buildAgent/work/a31d203e70e89f90/dist/target/wildfly-10.1.0.Final-SNAPSHOT/modules:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/modules, org.jboss.as.standalone, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas, -Djboss.server.base.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone, -Djboss.server.log.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/log, -Djboss.server.config.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/configuration, -Dts.wildfly.version=10.1.0.Final-SNAPSHOT, -c=standalone-xts.xml]
01:51:57,520 INFO  [org.xnio] (main) XNIO version 3.3.6.Final
01:51:57,538 INFO  [org.xnio.nio] (main) XNIO NIO Implementation Version 3.3.6.Final
TransformListener() : accepting requests on localhost:9091
01:51:57,614 INFO  [org.jboss.remoting] (main) JBoss Remoting version 4.0.18.Final
 [0m01:51:58,015 INFO  [org.jboss.modules] (main) JBoss Modules version 1.5.1.Final
 [0m [0m01:51:58,241 INFO  [org.jboss.msc] (main) JBoss MSC version 1.2.6.Final
 [0m [0m01:51:58,363 INFO  [org.jboss.as] (MSC service thread 1-6) WFLYSRV0049: WildFly Full 10.1.0.Final-SNAPSHOT (WildFly Core 2.2.0.CR2) starting
 [0m [0m01:51:59,765 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0039: Creating http management service using socket-binding (management-http)
 [0m [0m01:51:59,781 INFO  [org.xnio] (MSC service thread 1-4) XNIO version 3.3.6.Final
 [0m [0m01:51:59,789 INFO  [org.xnio.nio] (MSC service thread 1-4) XNIO NIO Implementation Version 3.3.6.Final
 [0m [0m01:51:59,822 INFO  [org.jboss.remoting] (MSC service thread 1-4) JBoss Remoting version 4.0.18.Final
 [0m [0m01:51:59,865 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 35) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 1.3)
 [0m [0m01:51:59,870 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 40) WFLYCLINF0001: Activating Infinispan subsystem.
 [0m [0m01:51:59,880 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 39) WFLYIO001: Worker 'default' has auto-configured to 8 core threads with 64 task threads based on your 4 available processors
 [0m [0m01:51:59,882 INFO  [org.wildfly.iiop.openjdk] (ServerService Thread Pool -- 41) WFLYIIOP0001: Activating IIOP Subsystem
 [0m [0m01:51:59,929 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-3) WFLYJCA0018: Started Driver service with driver-name = h2
 [0m [0m01:51:59,935 INFO  [org.jboss.as.connector] (MSC service thread 1-2) WFLYJCA0009: Starting JCA Subsystem (WildFly/IronJacamar 1.3.4.Final)
 [0m [33m01:51:59,998 WARN  [org.jboss.as.txn] (ServerService Thread Pool -- 59) WFLYTX0013: Node identifier property is set to the default value. Please make sure it is unique.
 [0m [0m01:52:00,013 INFO  [org.jboss.as.naming] (ServerService Thread Pool -- 51) WFLYNAM0001: Activating Naming Subsystem
 [0m [0m01:52:00,029 INFO  [org.jboss.as.jsf] (ServerService Thread Pool -- 47) WFLYJSF0007: Activated the following JSF Implementations: [main]
 [0m [0m01:52:00,040 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 58) WFLYSEC0002: Activating Security Subsystem
 [0m [0m01:52:00,056 INFO  [org.jboss.as.webservices] (ServerService Thread Pool -- 61) WFLYWS0002: Activating WebServices Extension
 [0m [0m01:52:00,121 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-8) WFLYUT0003: Undertow 1.3.21.Final starting
 [0m [0m01:52:00,152 INFO  [org.jboss.as.security] (MSC service thread 1-3) WFLYSEC0001: Current PicketBox version=4.9.6.Final
 [0m [0m01:52:00,160 INFO  [org.jboss.as.naming] (MSC service thread 1-1) WFLYNAM0003: Starting Naming Service
 [0m [0m01:52:00,161 INFO  [org.jboss.as.mail.extension] (MSC service thread 1-6) WFLYMAIL0001: Bound mail session [java:jboss/mail/Default]
 [0m [0m01:52:00,249 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 60) WFLYUT0014: Creating file handler for path '/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/welcome-content' with options [directory-listing: 'false', follow-symlink: 'false', case-sensitive: 'true', safe-symlink-paths: '[]']
 [0m [0m01:52:00,266 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0012: Started server default-server.
 [0m [0m01:52:00,271 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-1) WFLYUT0018: Host default-host starting


org.jboss.as.test.xts.wsat.client.ATTestCase: org.jboss.as.test.xts.annotation.client.TransactionalTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"transactional-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"transactional-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"transactional-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"transactional-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m01:52:06,218 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/data/content/e7/b3773a7875875989ebf228ee577dcd384d6e16/content
 [0m [0m01:52:06,225 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-3) WFLYSRV0027: Starting deployment of "transactional-test.war" (runtime-name: "transactional-test.war")
 [0m [31m01:52:06,343 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-4) MSC000001: Failed to start service jboss.deployment.unit."transactional-test.war".PARSE: org.jboss.msc.service.StartException in service jboss.deployment.unit."transactional-test.war".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment "transactional-test.war"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:154)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at org.wildfly.extension.undertow.deployment.WebFragmentParsingDeploymentProcessor.deploy(WebFragmentParsingDeploymentProcessor.java:87)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:147)
	... 5 more

 [0m [31m01:52:06,344 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 1) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "transactional-test.war")]) - failure description: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"transactional-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"transactional-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"transactional-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"transactional-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [31m01:52:06,344 ERROR [org.jboss.as.server] (management-handler-thread - 1) WFLYSRV0021: Deploy of deployment "transactional-test.war" was rolled back with the following failure message: 
{
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"transactional-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"transactional-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"transactional-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"transactional-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [0m01:52:06,360 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0028: Stopped deployment transactional-test.war (runtime-name: transactional-test.war) in 13ms
 [0m [0m01:52:06,361 INFO  [org.jboss.as.controller] (management-handler-thread - 1) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."transactional-test.war".PARSE



org.jboss.as.test.xts.wsat.client.ATTestCase: org.jboss.as.test.xts.annotation.compensationScoped.CompensationScopedTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m01:52:06,998 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/data/content/80/479a2f01182cbce67924e856d6873e337f3dd6/content
 [0m [0m01:52:07,001 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
 [0m [31m01:52:07,136 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-2) MSC000001: Failed to start service jboss.deployment.unit."test.war".PARSE: org.jboss.msc.service.StartException in service jboss.deployment.unit."test.war".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment "test.war"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:154)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at org.wildfly.extension.undertow.deployment.WebFragmentParsingDeploymentProcessor.deploy(WebFragmentParsingDeploymentProcessor.java:87)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:147)
	... 5 more

 [0m [31m01:52:07,137 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 3) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "test.war")]) - failure description: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [31m01:52:07,138 ERROR [org.jboss.as.server] (management-handler-thread - 3) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
{
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [0m01:52:07,150 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 4ms
 [0m [0m01:52:07,151 INFO  [org.jboss.as.controller] (management-handler-thread - 3) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."test.war".PARSE



org.jboss.as.test.xts.wsat.client.ATTestCase: org.jboss.as.test.xts.wsba.participantcompletion.client.BAParticipantCompletionTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-participantcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m01:52:07,752 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/data/content/f0/c32e872a9458f13e3c80f5ecbcb1492274b8a1/content
 [0m [0m01:52:07,755 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-7) WFLYSRV0027: Starting deployment of "wsba-participantcompletition-test.war" (runtime-name: "wsba-participantcompletition-test.war")
 [0m [31m01:52:07,951 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-3) MSC000001: Failed to start service jboss.deployment.unit."wsba-participantcompletition-test.war".PARSE: org.jboss.msc.service.StartException in service jboss.deployment.unit."wsba-participantcompletition-test.war".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment "wsba-participantcompletition-test.war"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:154)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at org.wildfly.extension.undertow.deployment.WebFragmentParsingDeploymentProcessor.deploy(WebFragmentParsingDeploymentProcessor.java:87)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:147)
	... 5 more

 [0m [31m01:52:07,954 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 1) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "wsba-participantcompletition-test.war")]) - failure description: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-participantcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [31m01:52:07,958 ERROR [org.jboss.as.server] (management-handler-thread - 1) WFLYSRV0021: Deploy of deployment "wsba-participantcompletition-test.war" was rolled back with the following failure message: 
{
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-participantcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-participantcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [0m01:52:07,971 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0028: Stopped deployment wsba-participantcompletition-test.war (runtime-name: wsba-participantcompletition-test.war) in 12ms
 [0m [0m01:52:07,971 INFO  [org.jboss.as.controller] (management-handler-thread - 1) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."wsba-participantcompletition-test.war".PARSE



org.jboss.as.test.xts.wsat.client.ATTestCase: org.jboss.as.test.xts.wsba.coordinatorcompletion.client.BACoordinatorCompletionTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-coordinatorcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m01:52:08,619 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/data/content/96/4e1061730afdc924934169e822f6baef71f7f5/content
 [0m [0m01:52:08,624 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "wsba-coordinatorcompletition-test.war" (runtime-name: "wsba-coordinatorcompletition-test.war")
 [0m [31m01:52:08,718 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-6) MSC000001: Failed to start service jboss.deployment.unit."wsba-coordinatorcompletition-test.war".PARSE: org.jboss.msc.service.StartException in service jboss.deployment.unit."wsba-coordinatorcompletition-test.war".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment "wsba-coordinatorcompletition-test.war"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:154)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at org.wildfly.extension.undertow.deployment.WebFragmentParsingDeploymentProcessor.deploy(WebFragmentParsingDeploymentProcessor.java:87)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:147)
	... 5 more

 [0m [31m01:52:08,724 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 2) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "wsba-coordinatorcompletition-test.war")]) - failure description: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-coordinatorcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [31m01:52:08,724 ERROR [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0021: Deploy of deployment "wsba-coordinatorcompletition-test.war" was rolled back with the following failure message: 
{
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsba-coordinatorcompletition-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsba-coordinatorcompletition-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [0m01:52:08,741 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0028: Stopped deployment wsba-coordinatorcompletition-test.war (runtime-name: wsba-coordinatorcompletition-test.war) in 15ms
 [0m [0m01:52:08,741 INFO  [org.jboss.as.controller] (management-handler-thread - 2) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."wsba-coordinatorcompletition-test.war".PARSE



org.jboss.as.test.xts.wsat.client.ATTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsat-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsat-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsat-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsat-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m01:52:09,608 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/xts/target/jbossas/standalone/data/content/b6/837ee5f6b519ef5c3decf026068ff6347cbc45/content
 [0m [0m01:52:09,619 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "wsat-test.war" (runtime-name: "wsat-test.war")
 [0m [31m01:52:09,812 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-4) MSC000001: Failed to start service jboss.deployment.unit."wsat-test.war".PARSE: org.jboss.msc.service.StartException in service jboss.deployment.unit."wsat-test.war".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment "wsat-test.war"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:154)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at org.wildfly.extension.undertow.deployment.WebFragmentParsingDeploymentProcessor.deploy(WebFragmentParsingDeploymentProcessor.java:87)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:147)
	... 5 more

 [0m [31m01:52:09,814 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 4) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "wsat-test.war")]) - failure description: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsat-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsat-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsat-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsat-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [31m01:52:09,814 ERROR [org.jboss.as.server] (management-handler-thread - 4) WFLYSRV0021: Deploy of deployment "wsat-test.war" was rolled back with the following failure message: 
{
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"wsat-test.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"wsat-test.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"wsat-test.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"wsat-test.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
 [0m [0m01:52:09,822 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0028: Stopped deployment wsat-test.war (runtime-name: wsat-test.war) in 7ms
 [0m [0m01:52:09,822 INFO  [org.jboss.as.controller] (management-handler-thread - 4) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."wsat-test.war".PARSE



org.wildfly.test.extension.rts.ProvidersTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {
    "WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"test-deployment.war\".PARSE" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"test-deployment.war\".PARSE: WFLYSRV0153: Failed to process phase PARSE of deployment \"test-deployment.war\"
    Caused by: java.lang.NullPointerException"},
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.deployment.unit.\"test-deployment.war\".PARSE"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => undefined
}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
01:52:14,011 WARNING [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Bundles path is deprecated and no longer used.
01:52:14,012 INFO  [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Starting container with: [/usr/java/latest/bin/java, -D[Standalone], -Dorg.jboss.ejb.client.wildfly-testsuite-hack=true, -Xmx512m, -XX:MetaspaceSize=128m, -Djboss.dist=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/../../../dist/target/wildfly-10.1.0.Final-SNAPSHOT, -Djava.net.preferIPv4Stack=false, -Djava.net.preferIPv6Addresses=true, -server, -Dts.timeout.factor=100, -Dnode0=::1, -Dnode1=::1, -Dmcast=ff13::1, -Dmcast.ttl=0, -Djbossas.ts.submodule.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts, -Djbossas.ts.integ.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/.., -Djbossas.ts.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/../.., -Djbossas.project.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/../../.., -Djboss.inst=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas, -ea, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas, -Dorg.jboss.boot.log.file=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/standalone/log/server.log, -Dlogging.configuration=file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/standalone/configuration/logging.properties, -jar, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/jboss-modules.jar, -mp, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/modules:/opt/buildAgent/work/a31d203e70e89f90/dist/target/wildfly-10.1.0.Final-SNAPSHOT/modules:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/modules, org.jboss.as.standalone, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas, -Djboss.server.base.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/standalone, -Djboss.server.log.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/standalone/log, -Djboss.server.config.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/standalone/configuration, -Dts.wildfly.version=10.1.0.Final-SNAPSHOT, -c=standalone-rts.xml]
01:52:14,097 INFO  [org.xnio] (main) XNIO version 3.3.6.Final
01:52:14,115 INFO  [org.xnio.nio] (main) XNIO NIO Implementation Version 3.3.6.Final
01:52:14,190 INFO  [org.jboss.remoting] (main) JBoss Remoting version 4.0.18.Final
 [0m01:52:14,449 INFO  [org.jboss.modules] (main) JBoss Modules version 1.5.1.Final
 [0m [0m01:52:14,630 INFO  [org.jboss.msc] (main) JBoss MSC version 1.2.6.Final
 [0m [0m01:52:14,710 INFO  [org.jboss.as] (MSC service thread 1-6) WFLYSRV0049: WildFly Full 10.1.0.Final-SNAPSHOT (WildFly Core 2.2.0.CR2) starting
 [0m [0m01:52:16,285 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0039: Creating http management service using socket-binding (management-http)
 [0m [0m01:52:16,301 INFO  [org.xnio] (MSC service thread 1-8) XNIO version 3.3.6.Final
 [0m [0m01:52:16,336 INFO  [org.xnio.nio] (MSC service thread 1-8) XNIO NIO Implementation Version 3.3.6.Final
 [0m [0m01:52:16,379 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 42) WFLYCLINF0001: Activating Infinispan subsystem.
 [0m [0m01:52:16,398 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 41) WFLYIO001: Worker 'default' has auto-configured to 8 core threads with 64 task threads based on your 4 available processors
 [0m [0m01:52:16,398 INFO  [org.wildfly.iiop.openjdk] (ServerService Thread Pool -- 43) WFLYIIOP0001: Activating IIOP Subsystem
 [0m [0m01:52:16,426 INFO  [org.jboss.as.naming] (ServerService Thread Pool -- 53) WFLYNAM0001: Activating Naming Subsystem
 [0m [0m01:52:16,433 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 60) WFLYSEC0002: Activating Security Subsystem
 [0m [0m01:52:16,435 INFO  [org.jboss.as.security] (MSC service thread 1-6) WFLYSEC0001: Current PicketBox version=4.9.6.Final
 [0m [33m01:52:16,445 WARN  [org.jboss.as.txn] (ServerService Thread Pool -- 61) WFLYTX0013: Node identifier property is set to the default value. Please make sure it is unique.
 [0m [0m01:52:16,459 INFO  [org.jboss.as.webservices] (ServerService Thread Pool -- 63) WFLYWS0002: Activating WebServices Extension
 [0m [0m01:52:16,462 INFO  [org.jboss.as.jsf] (ServerService Thread Pool -- 49) WFLYJSF0007: Activated the following JSF Implementations: [main]
 [0m [0m01:52:16,494 INFO  [org.jboss.remoting] (MSC service thread 1-8) JBoss Remoting version 4.0.18.Final
 [0m [0m01:52:16,502 INFO  [org.jboss.as.connector] (MSC service thread 1-3) WFLYJCA0009: Starting JCA Subsystem (WildFly/IronJacamar 1.3.4.Final)
 [0m [0m01:52:16,502 INFO  [org.jboss.as.naming] (MSC service thread 1-4) WFLYNAM0003: Starting Naming Service
 [0m [0m01:52:16,515 INFO  [org.jboss.as.mail.extension] (MSC service thread 1-1) WFLYMAIL0001: Bound mail session [java:jboss/mail/Default]
 [0m [0m01:52:16,596 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 37) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 1.3)
 [0m [0m01:52:16,598 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0018: Started Driver service with driver-name = h2
 [0m [0m01:52:16,633 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-3) WFLYUT0003: Undertow 1.3.21.Final starting
 [0m [0m01:52:16,871 INFO  [org.jboss.as.ejb3] (MSC service thread 1-4) WFLYEJB0481: Strict pool slsb-strict-max-pool is using a max instance size of 64 (per class), which is derived from thread worker pool sizing.
 [0m [0m01:52:16,872 INFO  [org.jboss.as.ejb3] (MSC service thread 1-3) WFLYEJB0482: Strict pool mdb-strict-max-pool is using a max instance size of 16 (per class), which is derived from the number of CPUs on this host.
 [0m [0m01:52:16,938 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 62) WFLYUT0014: Creating file handler for path '/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/rts/target/jbossas/welcome-content' with options [directory-listing: 'false', follow-symlink: 'false', case-sensitive: 'true', safe-symlink-paths: '[]']
 [0m [0m01:52:16,945 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0012: Started server default-server.


```

--- a/undertow/src/main/java/org/wildfly/extension/undertow/deployment/WebFragmentParsingDeploymentProcessor.java
+++ b/undertow/src/main/java/org/wildfly/extension/undertow/deployment/WebFragmentParsingDeploymentProcessor.java
@@ -81,7 +81,11 @@ public class WebFragmentParsingDeploymentProcessor implements DeploymentUnitProc
                         inputFactory.setXMLResolver(NoopXMLResolver.create());
                         XMLStreamReader xmlReader = inputFactory.createXMLStreamReader(is);
 
-                        webFragments.put(resourceRoot.getRootName(), WebFragmentMetaDataParser.parse(xmlReader, SpecDescriptorPropertyReplacement.propertyReplacer(deploymentUnit)));
+                        WebFragmentMetaData webFragmentMetaData = WebFragmentMetaDataParser.parse(xmlReader, SpecDescriptorPropertyReplacement.propertyReplacer(deploymentUnit));
+                        webFragments.put(resourceRoot.getRootName(), webFragmentMetaData);
+                        /*Log message to inform that distributable is not set in web-fragment.xml while it is set in web.xml*/
+                        if (warMetaData.getWebMetaData() != null && warMetaData.getWebMetaData().getDistributable()!= null && webFragmentMetaData.getDistributable() == null)
+                            UndertowLogger.ROOT_LOGGER.distributableDisabledInFragmentXml(deploymentUnit.getName(),resourceRoot.getRootName());
                     } catch (XMLStreamException e) {
                         throw new DeploymentUnitProcessingException(UndertowLogger.ROOT_LOGGER.failToParseXMLDescriptor(webFragment.toString(), e.getLocation().getLineNumber(), e.getLocation().getColumnNumber()));
                     } catch (IOException e) {

--- a/undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java
+++ b/undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java
@@ -342,4 +342,8 @@ public interface UndertowLogger extends BasicLogger {
     @LogMessage(level = WARN)
     @Message(id = 80, value = "Valves are no longer supported, %s is not activated.")
     void unsupportedValveFeature(String valve);
+
+    @LogMessage(level = WARN)
+    @Message(id = 81, value = "The deployment %s will not be distributable because this feature is disabled in web-fragment.xml of the module %s.")
+    void distributableDisabledInFragmentXml(String deployment, String module);
 }

